src/pvrclient-nextpvr.h

```diff
@@ -112,7 +124,7 @@ class cPVRClientNextPVR : P8PLATFORM::CThread
   int GetRecordingLastPlayedPosition(const PVR_RECORDING &recording);
   PVR_ERROR GetRecordingEdl(const PVR_RECORDING& recording, PVR_EDL_ENTRY[], int *size);
   PVR_ERROR GetRecordingStreamProperties(const PVR_RECORDING*, PVR_NAMED_VALUE*, unsigned int*);
-  bool UpdatePvrRecording(TiXmlElement* pRecordingNode, PVR_RECORDING *tag);
+  bool UpdatePvrRecording(TiXmlElement* pRecordingNode, PVR_RECORDING *tag, const std::string title, bool flatten);
```

Comment:
`const std::string& title`

Reply:
OK I don't know why but I will change it.